composer.json

```diff
@@ -89,7 +89,16 @@
     "extra": {
         "installer-name": "Framework",
         "neos": {
-            "warning": "AUTOGENERATED FILE, ONLY MODIFY THE .composer.json IN THIS DIRECTORY AND RUN THE MERGE SCRIPT."
+            "warning": "AUTOGENERATED FILE, ONLY MODIFY THE .composer.json IN THIS DIRECTORY AND RUN THE MERGE SCRIPT.",
+            "package-key": [
```

Comment:
It wouldn't matter much and I guess as such I prefer to leave it in as to avoid manual edititing of this file.